digiroad2-api-common/src/main/scala/fi/liikennevirasto/digiroad2/Digiroad2Context.scala

```diff
@@ -71,6 +71,31 @@ class ValluTerminalActor(massTransitStopService: MassTransitStopService) extends
   }
 }
 
+class LanesUpdater(laneService: LaneService) extends Actor {
+  def receive = {
+    case x: LaneFiller.ChangeSet => persistLaneChanges(x )
+    case _            => println("LinearAssetUpdater: Received unknown message")
+  }
+
+  def persistLaneChanges(changeSet: LaneFiller.ChangeSet) {
+    laneService.updateChangeSet(changeSet)
+  }
+}
+
+class LanesGenerator[T](laneService: LaneService) extends Actor {
+  def receive = {
+    case persistedLanes: Seq[T] =>  laneService.generateLanes(persistedLanes.asInstanceOf[Seq[PersistedLane]])
+    case _ => println("LaneAssetGenerator: Received unknown message")
+  }
+}
```

Comment:
This LaneGenarator is just a duplicate behaviour as LaneSaveProjected

Reply:
LaneGenerator deleted now aggregated with the old LaneSaveProjected